common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -121,6 +121,19 @@ private static async Task Run(Type testType, PerfOptions options)
                             await RunTestsAsync(tests, options, title);
                         }
                     }
+                    catch (AggregateException ae)
+                    {
+                        foreach (Exception e in ae.InnerExceptions)
+                        {
+                            Console.WriteLine($"Exception: {e}");
+                        }
+                        throw;
+                    }
+                    catch (Exception e)
+                    {
+                        Console.WriteLine($"Exception: '{e}");
```

Comment:
nit:  134 has an opening single quote before the value of `e` and no closing quote.   Line 128 has no single quotes around `e`.    We should probably normalize that.